plugins/webp-uploads/rest-api.php

```diff
@@ -49,6 +49,6 @@ function webp_uploads_update_rest_attachment( WP_REST_Response $response, WP_Pos
 		unset( $data['media_details']['sources'] );
 	}
 
-	return rest_ensure_response( $data );
```

Comment:
The `rest_ensure_response()` function returns a `WP_REST_Response` _or_ a `WP_Error`. Since we are passing back `$data` and we know it is not a `WP_Error`, we can explicitly return a `WP_REST_Response`.

Reply:
So ething that can be contributed upstream actually in the wp stubs repo

Reply:
How? This `webp_uploads_update_rest_attachment()` returns a `WP_REST_Response`. But `rest_ensure_response()` returns either a `WP_REST_Response` or a `WP_Error`. Since we're never returning a `WP_Error` we don't need `rest_ensure_response()`?

Reply:
Sorry, what I meant is a PR can be contributed to the `wordpress-stubs` repository that adds `@phpstan-return ($response is WP_Error ? WP_Error : WP_REST_Response)` to `rest_ensure_response`

Then you theoretically don't need to make this change here.

Hope that makes sense.

Reply:
Oh, ok. I'll have to double check this then as to the error I was getting.

Reply:
OK, I see. So while `rest_ensure_response()` _should_ work, it doesn't yet work without a patch to `wordpress-stubs`.

Currently it gives the following with `return rest_ensure_response( $data )`:

```
 ------ ---------------------------------------------------------------------------------------------------------------------- 
  Line   plugins/webp-uploads/rest-api.php                                                                                     
 ------ ---------------------------------------------------------------------------------------------------------------------- 
  52     Function webp_uploads_update_rest_attachment() should return WP_REST_Response but returns WP_Error|WP_REST_Response.  
 ------ ---------------------------------------------------------------------------------------------------------------------- 
```

Reply:
Yeah that is expected without the patch. Wanna contribute a PR? :)

Reply:
Hummm. I tried patching wordpress-stubs locally to make that change, but it didn't work 😕

Reply:
Fixed in https://github.com/php-stubs/wordpress-stubs/pull/177